hera_mc/part_connect.py

```diff
@@ -292,6 +292,68 @@ def get_part_revisions(hpn, session=None):
     return revisions
 
 
+class Dubitable(MCDeclarativeBase):
+    """
+    A table to track the dubitable antennas.
+
+    start_gpstime:  start time of list
+    stop_gpstime:  stop time of list
+    ant_list:  list of "unapproved" antennas
```

Comment:
This needs to be much more explicit about the contents of `ant_list`. Below I see that it is comma-separated, but that's not specified here. How are we expressing antennas? Prefixed part numbers? Simple integers? Etc. Can you add a cross-reference to the CM docs about what precise antenna identifier will be used here?